pkg/cli/proxy_get.go

```diff
@@ -0,0 +1,71 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"net/http"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+	"github.com/openservicemesh/osm/pkg/k8s"
+)
+
+func GetEnvoyProxyConfig(clientSet kubernetes.Interface, config *rest.Config, namespace string, podName string, localPort uint16, query string) (string, error) {
```

Comment:
I propose we return `[]byte` instead of `string`. Another function could take the byte slice and unmarshal to xDS structs ([ConfigDump proto](https://www.envoyproxy.io/docs/envoy/latest/api-v3/admin/v3/config_dump.proto) in this case)